### PR TITLE
Install theme in window

### PR DESCRIPTION
**Pull request by sarthak-marwaha**

### CODE REVIEW

File: src/foam/u2/theme/StandaloneTheme.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  requires: [
+    'foam.u2.theme.ThemeGlyphs'
+  ],
+
+  properties: [
+    {
+      class: 'String',
+      name: 'id',
+      includeInDigest: true,
```
Comment (kgrgreer) [inline src/foam/u2/theme/StandaloneTheme.js]: Is this needed?

File: src/foam/u2/theme/StandaloneTheme.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      section: 'infoSection',
+      createVisibility: 'HIDDEN',
```
Comment (kgrgreer) [inline src/foam/u2/theme/StandaloneTheme.js]: Is visibility information needed?

File: src/foam/u2/theme/StandaloneTheme.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    {
+      class: 'String',
+      name: 'name',
+      writePermissionRequired: true
+    },
+    {
+      class: 'Image',
+      name: 'logo',
+      documentation: 'The logo to display in the application.',
+      displayWidth: 60,
```
Comment (kgrgreer) [inline src/foam/u2/theme/StandaloneTheme.js]: needed?

File: src/foam/u2/theme/StandaloneTheme.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      view: {
```
Comment (kgrgreer) [inline src/foam/u2/theme/StandaloneTheme.js]: needed?

File: src/foam/u2/theme/StandaloneTheme.js (hunk 1 above)
Comment (jlhughes) [reply src/foam/u2/theme/StandaloneTheme.js]: includeInDigest of the 'id' is not necessary.

File: src/foam/u2/theme/StandaloneTheme.js (hunk 2 above)
Comment (jlhughes) [reply src/foam/u2/theme/StandaloneTheme.js]: This ensures the id is auto generated.

File: src/foam/u2/theme/StandaloneTheme.js (hunk 1 above)
Comment (kgrgreer) [reply src/foam/u2/theme/StandaloneTheme.js]: But there will only be one of these objects in the system and it won't be stored in a DAO. Correct Sarthak?

File: src/foam/u2/theme/StandaloneTheme.js (hunk 4 above)
Comment (kgrgreer) [reply src/foam/u2/theme/StandaloneTheme.js]: This object will never be shown in the GUI, correct?

File: src/foam/u2/theme/StandaloneTheme.js (hunk 1 above)
Comment (sarthak-marwaha) [reply src/foam/u2/theme/StandaloneTheme.js]: Yes, will update

File: src/foam/u2/theme/StandaloneTheme.js (hunk 2 above)
Comment (sarthak-marwaha) [reply src/foam/u2/theme/StandaloneTheme.js]: Not in this case, this theme object never goes to the DAO, this is a theme for when we dont use nanos

**Pull request merged**